### PR TITLE
Refs #32398 - Pass router history to tabs

### PR DESCRIPTION
**Pull request by xprazak2**
Needed for proper navigation from tabs, based on https://github.com/theforeman/foreman/pull/8670/files#r673134856

### CODE REVIEW

**Comment by MariaAga** [comment]
@xprazak2  https://github.com/theforeman/foreman/pull/8738 was merged instead of  https://github.com/theforeman/foreman/pull/8670 So where is this needed?

**Comment by xprazak2** [comment]
@MariaAga, this is something that was in POC but was omited in the version that was merged. It is needed to create [links](https://github.com/theforeman/foreman_ansible/pull/419#discussion_r703458457) with correct path.

File: webpack/assets/javascripts/react_app/components/HostDetails/index.js [hunk 1]
```diff
@@ -127,6 +128,7 @@ const HostDetails = ({
             hostName={id}
             status={status}
             tabs={tabs}
+            router={history}
```
Comment (MariaAga) [inline webpack/assets/javascripts/react_app/components/HostDetails/index.js:131]: Why the name change here?

Comment (amirfefer) [reply webpack/assets/javascripts/react_app/components/HostDetails/index.js:131]: I guess this change is made to distinguish between the main router and hash-router history.

**Review by amirfefer — APPROVED** [review]
LGTM 👍 
Thanks @xprazak2

File: webpack/assets/javascripts/react_app/components/HostDetails/index.js (hunk 1 above)
Comment (MariaAga) [reply webpack/assets/javascripts/react_app/components/HostDetails/index.js:131]: Then should it be called `hashRouter` instead?

Comment (amirfefer) [reply webpack/assets/javascripts/react_app/components/HostDetails/index.js:131]: This history object represents the main router's history, not the `hashRouter` one.

**Pull request merged**